src/rules/mod.rs

```diff
@@ -33,6 +33,7 @@ mod no_func_assign;
 mod no_misused_new;
 mod no_namespace;
 mod no_new_symbol;
+mod no_non_null_assertion;
```

Comment:
can you add this rule to the Readme too, please?

Reply:
Yes I have. Thanks